tests/conftest.py

```diff
@@ -175,6 +175,40 @@ def pytest_addoption(parser):
             "may be insecure! Default: False"
         ),
     )
+    test_selection_group.addoption(
+        "--core",
+        "--core-tests",
+        dest="core",
+        action="store_true",
+        default=False,
+        help=(
+            "Run salt-core tests. These tests test the engine of salt! "
+            "Default: False"
+        ),
+    )
+    test_selection_group.addoption(
```

Comment:
Why do we need a fast group, when we already have a slow group. The tests should run the fast tests automatically and then also run the slow tests if `--run-slow` is specified.

Reply:
it so if you want to exclude fast tests. For examples your fast test all pass, but you want to test your slow test. You dont want to wait for another 15 mins for the fast tests to run again.

Reply:
core test are slow test but that also just test the low level stuff of salt like pillar or beacon

Reply:
Do we have to decorate every single test that is not a slow or core test?

Reply:
no, pytest checks if !core_test or !slow_test -> treat test as fast_test